yaksh/models.py

```diff
@@ -539,6 +539,18 @@ def get_answerpaper_status(self, user, course):
             status = "not attempted"
         return status
 
+    def get_answerpaper_passing_status(self, user, course):
+        try:
+            qp = self.questionpaper_set.get().id
+        except QuestionPaper.DoesNotExist:
+            qp = None
+        ans_ppr = AnswerPaper.objects.filter(
+            user=user, course=course, question_paper=qp
+        ).order_by("-attempt_number")
```

Comment:
Should this be order by percentage?
As there can be a case that in 2 attempts, a user passed in the first and failed in the second!
What are your thoughts?